portfolio/src/main/webapp/calendar.js

```diff
@@ -36,7 +38,11 @@ function createNewCalendarEvent() {
   const endTime =
       getTimeObject(document.getElementById('new-event-end-time').value);
 
-  const eventName = document.getElementById('new-event-name').value;
+  // If user enters an empty event name, sets the name to a default string.
+  let eventName = document.getElementById('new-event-name').value;
+  if (!eventName) {
```

Comment:
Maybe it's simpler to do,

let eventName = document.... || 'New Event';

Reply:
Updated.